src/app/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styles from './Slider.module.css';
+import Typography from '../Typography/Typography';
+
+export type SliderProps = {
+  size: 'minutes' | 'hours';
```

Comment:
Wouldn't it be much cooler if you could just choose whatever you want? I mean, it's not necessary, but it would be possible.